modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.tsx

```diff
@@ -641,6 +663,17 @@ const FrontendDataSetContent = ({
 			return;
 		}
 
+		const clientExtensionFiltersLoading = filters.some(
+			(filter: any) =>
+				filter.clientExtensionFilterURL &&
+				!filter.clientExtensionFilterImplementation &&
+				!filter.clientExtensionFilterResolutionError
+		);
+
+		if (clientExtensionFiltersLoading) {
+			return;
+		}
+
```

Comment:
shall we `setDataLoading(true);` before returning, in case of having unresolved CX?

would we like to apply this pattern also for cell renderers cx?